checklists/sap_checklist.en.json

```diff
@@ -1118,7 +1136,8 @@
             "guid": "85e2213a-ce7b-4b12-8f7c-95f06e154e3a",
             "severity": "High",
             "training": "https://learn.microsoft.com/training/paths/azure-fundamentals-describe-azure-architecture-services/?source=recommendations",
-            "link": "https://learn.microsoft.com/azure/virtual-network/accelerated-networking-overview?tabs=redhat"
+            "link": "https://learn.microsoft.com/azure/virtual-network/accelerated-networking-overview?tabs=redhat",
+            "graph": "Resources | where type =~ "Microsoft.Network/NetworkInterfaces" | where properties.enableAcceleratedNetworking == "false" | project name, subscriptionId, properties.enableAcceleratedNetworking"
```

Comment:
You need to replace the double quotes (`"`) inside of the query by single quotes (`'`), othewise the JSON breaks.